homeassistant/components/notify_mqtt/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "notify_mqtt",
```

Comment:
This should be a platform under the mqtt integration, not its own integration.

Reply:
Done.